consensus/polybft/signer/utils.go

```diff
@@ -17,18 +17,18 @@ var (
 )
 
 const (
-	DomainValidatorSetString      = "DOMAIN_CHILD_VALIDATOR_SET"
-	DomainCheckpointManagerString = "DOMAIN_CHECKPOINT_MANAGER"
-	DomainCommonSigningString     = "DOMAIN_COMMON_SIGNING"
-	DomainStateReceiverString     = "DOMAIN_STATE_RECEIVER"
+	DomainValidatorSetString  = "DOMAIN_CHILD_VALIDATOR_SET"
+	DomainBlockMetaDataString = "DOMAIN_BRIDGE_META"
```

Comment:
This is what contracts expect:

```suggestion
	DomainBridgeString = "DOMAIN_BRIDGE"
```